Sleep Tracker/Sleep Tracker/View/SleepSetupView/Subviews/NoAlarmLabel.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+
+/// A view that shows information about the disabled alarm clock.
+struct NoAlarmLabel: View {
```

Comment:
nit:

```suggestion
struct NoAlarmLabelView: View {
```

(if you had such agreement)

Reply:
fixed